index.js

```diff
@@ -11,6 +28,22 @@ module.exports = {
     };
   },
 
+  config: function(/* environment, appConfig */) {
+    // do nothing unless running `ember fastboot` command
+    if (!process.env.EMBER_CLI_FASTBOOT) { return {}; }
+
+    return {
+      autoRun: false,
+      storeConfigInMeta: false,
+      EmberENV: {
+        FEATURES: { 'ember-application-visit': true }
```

Comment:
Hmm, from what I read, the whole thing is deep merged with lodash. I will confirm!